code/src/App.js

```diff
@@ -1,12 +1,43 @@
-import React from 'react'
-import data from './data.json'
-
-console.log(data)
+import React from "react";
+import data from "./data.json";
+import Article from "./components/Article";
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <>
+      <h1>New Albums and singles</h1>
+      <div className="container">
+        {data.albums.items.map((album) => (
+          <Article
+            key={album.id}
+            title={album.name}
+            titleClick={album.external_urls.spotify}
+            img={album.images[1].url}
+            artist={album.artists.map((item, index) => (
+              <a
+                key={item.id}
+                href={item.external_urls.spotify}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                {index ? "," : ""} {item.name}
+              </a>
+            ))}
+          />
+        ))}
+      </div>
+    </>
+  );
+};
+
+/* 
+
+TODO
```

Comment:
Suggestion to remove the todo list from the final code (if you consider yourself done already that is)!